controller/tests/array_action/ds8k/test_array_mediator_ds8k.py

```diff
@@ -418,16 +418,21 @@ def test_get_snapshot_get_fcrel_not_exist_raise_error(self):
         self.client_mock.get_volumes_by_pool.return_value = [target_vol]
         self.client_mock.get_flashcopies_by_volume.return_value = []
 
-        with self.assertRaises(array_errors.SnapshotNameBelongsToVolumeError):
+        with self.assertRaises(array_errors.ExpectedSnapshotButFoundVolumeError):
             self.array.get_snapshot("test_name", pool_id=self.volume_response.pool)
 
     def test_get_snapshot_success(self):
         target_vol = self._get_volume_with_flashcopy_relationship()
         self.client_mock.get_volumes_by_pool.return_value = [target_vol]
+        flashcopy_response = copy.deepcopy(self.flashcopy_response)
```

Comment:
why the `deepcopy`?

Reply:
so it could be different element.. not a reference

Reply:
I know what it does. *why* is it needed?

Reply:
so no need to right the flashcopy again..

Reply:
done (removed the deepcopy

Reply:
to minimize the chance of this happening again in the future, please also remove it in:
`test_delete_volume_with_flashcopies_as_source_and_target_failed`
`test_delete_volume_with_flashcopy_as_target_success`
`test_create_snapshot_not_valid`
`_prepare_mocks_for_copy_to_existing_volume`
and other redundant places if you find any.

Reply:
done